aws/customizations/workspace/solution_accelerators/security_analysis_tool/aws/provider.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    databricks = {
+      source = "databricks/databricks"
+    }
+  }
+}
+
+module "common" {
```

Comment:
Minor beef with this being called common and the source folder being called that, but I'm not familiar with the TF of SAT to understand why it would be called common, seems unnecessarily ambiguous

Reply:
yeah - this will be to raise directly with SAT